earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -147,12 +157,8 @@ install_or_upgrade_asset_root()
         chown -R "$GEFUSIONUSER:$GEGROUP" "$ASSET_ROOT"
     else
         # upgrade asset root -- if this is a master
-        if [ "$IS_SLAVE" = false ]; then
-            # TODO: Verify this logic -- this is what is defined in the
-            # installer documentation, but needs confirmation
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gegroup_existed NEW_GEFUSIONUSER
-            keyvalue_file_get "$GEE_INSTALL_KV_PATH" gefusionuser_existed NEW_GEFUSIONUSER
-            if [ "$NEW_GEGROUP" = true ] || [ "$NEW_GEFUSIONUSER" = true ]; then
+        if [ "$IS_SLAVE" = "false" ]; then
+            if [ "$NEW_INSTALL" = "false" ] ; then
```

Comment:
What I'm saying is that this is modifying the existing behavior by no longer testing if `gegroup` was created.

I don't think the question is whether `gegroup` exists after it is created by installing `opengee-common`.  What I understand the code is trying to do is:

* If asset data existed before `gegroup` or `gefusionuser` were created in the install scripts, change file ownership of the data.
* If `gegroup` and `gefusionuser` already existed before starting an installation, assume ownership is correct, and don't change it.

This change skips the test for whether `gegroup` existed.  I think we could write the tests better, but I'm wondering if it's a good idea to skip one of them.

Reply:
And what I am saying is I deduce this from rather different means, based on both the rpm install/upgrade state, and also on how NEW_INSTALL gets set if there was a prior non-rpm install.

Reply:
The test for 'a prior non-rpm install' seems to be whether `/etc/init.d/gefusion` exists.  So, if a user had an old version of GEE and Fusion with some data, uninstalled them, and installed RPMs, that assert root owner update will no longer happen, and Fusion would complain.  (I don't think it would work to install RPMs while the old version is installed anyway, at least until we implement the upgrade scripts from old versions.)

This is not very hard to fix, so I don't think it's a great problem, but we also don't have to introduce it.

Reply:
I believe we already had this problem possible in the older script.

Reply:
:stuck_out_tongue: The older script wouldn't have that problem, since it relied on whether the group and user existed, not on whether `/etc/init.d/gefusion` existed. Anyway, we can tell an administrator how to fix that after an upgrade.

Reply:
Could you add a small Markdown file on upgrading that includes the command to update an asset root from an old install?

Reply:
If this is only a matter of permissions and assuring the group/ownership match up, it might be easier to do a chgrp/chown...